engine/installation/linux/debian.md

```diff
@@ -287,13 +312,14 @@ installing Docker.
     Use this command to install the latest version of Docker:
 
     ```bash
-    $ sudo apt-get install docker
+    $ sudo apt-get install docker-engine
```

Comment:
You should definitely note that you are not getting `docker-ce` but an older version. @andrewhsu might have more info on the maintenance aspect. This would be  significant downside to using this script.